src/wise_utils.jl

```diff
@@ -0,0 +1,14 @@
+function wise_params(band_id::Int64)
```

Comment:
Better not to use a function if it's just doing a lookup. Just declare the values your returning in global scope with `const`.